source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,475 @@
+---
+title: Front End Performance
+description: Learn how to ace an online speed test with Pantheon's Global CDN.
+tags: [siteintegrations]
+categories: []
+type: guide
+image: CDN-speedTest-docs-guide
+permalink: docs/guides/:basename/
+contributors:
+  - tessak22
+  - stevector
+  - ataylorme
+  - rachelwhitton
+  - dwayne
+  - dgorton
+  - dts
+---
+Nobody likes a slow site. In this guide we’ll explore how to speed up front end performance to ensure that visitors to your site stay engaged and have the best experience.  
+
+<div class="enablement">
+ <a href="https://pantheon.io/agencies/learn-pantheon?docs"><h4 class="info">Agency DevOps Training</h4></a>
+ <p>Learn industry best practices for caching, how to take advantage of them on the platform, and troubleshooting common issues with help from the experts at Pantheon.</p>
+</div>
+
+
+There are many different ways to measure page speed and performance. This guide will be tuning for the metrics set forth by [Google's mobile speed test](https://testmysite.thinkwithgoogle.com/){.external}:
+
+- Reduce Server Response Time
```

Comment:
@rachelwhitton, did we want this list to line up exactly with the major headings in the sidebar? They no longer do.

Reply:
Hmm, I think this list should use the categories from Google's results. Otherwise I prefer we ditch the list entirely because matching it to the major headings is redundant to the fixed TOC we have on the right.

Reply:
k, let's leave it as is.